sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -223,6 +223,30 @@ static const struct snd_soc_acpi_link_adr adl_sdw_rt1316_link2_rt714_link0[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_adr_device mx8373_2_adr[] = {
+        {
+                .adr = 0x000223019F837300,
+                .num_endpoints = 1,
+                .endpoints = &spk_l_endpoint,
+                .name_prefix = "Right"
```

Comment:
l_endpoint but prefix is "Right".. reversed?

Reply:
Follwed reference from TGL , but looks not good to me as well
https://github.com/thesofproject/linux/blob/9101539a195b6d7a4554b74d89af9d09ef52afd7/sound/soc/intel/common/soc-acpi-intel-tgl-match.c#L136

Reply:
Resolved